test/functional/testlib.bash

```diff
@@ -2693,3 +2705,40 @@ assert_files_not_equal() {
 	fi
 
 }
+
+# Displays a menu of functions that can be used from within a test. Functions considered
+# "internal" are not displayed.
+# Functions are not included in the list by default, to mark a function to be included in
+# the help menu, add a comment immediatelly after the function opening bracket with either
+# the keyword swupd_function or assertion depending on the type of function.
+# Parameters:
+# - prefix: a prefix that could be used to filter functions
+testlib() {
+
+	local prefix=$1
+	local funcs
+	local assertions
+
+	funcs=$(grep "$prefix.*() { [#] swupd_function" "$FUNC_DIR"/testlib.bash | cut -f1 -d "(")
+	assertions=$(grep "$prefix.*() { [#] assertion" "$FUNC_DIR"/testlib.bash | cut -f1 -d "(")
+	echo ""
+	if [ -z "$prefix" ]; then
```

Comment:
I prefer to leave the header even if prefix is set.

Reply:
I will fix that @otaviobp